plugin/secrets_access_token.go

```diff
@@ -182,3 +104,63 @@ func (tg *TokenGenerator) getAccessToken(ctx context.Context, iamAdmin *iam.Serv
 	}
 	return tkn, err
 }
+
+const deprecationWarning = `
+We have recently changed the behavior of the gcp/token/ endpoint to no longer generate leases. 
+This is due to limitations of the GCP API, as service account OAuth access tokens cannot be revoked.
+This access_token secret and associated lease were created by an older version of the GCP secrets engine
+and will be cleaned up now. Note that there is the chance that this access_token, if not already expired, 
+will still be valid up to one hour. 
+`
+
+const pathTokenHelpSyn = `Generate an OAuth2 access token under a specific role set.`
+const pathTokenHelpDesc = `
+This path will generate a new OAuth2 access token for accessing GCP APIs.
+A role set, binding IAM roles to specific GCP resources, will be specified 
+by name - for example, if this backend is mounted at "gcp",
+then "gcp/token/deploy" would generate tokens for the "deploy" role set.
+
+On the backend, each roleset is associated with a service account. 
+The token will be associated with this service account. Tokens have a 
+short-term lease (1-hour) associated with them but cannot be renewed.
+
+NOTE: As of Vault 0.11.2, we have changed the behavior of the token/ endpoint.
```

Comment:
I don't think we should have this in help text. The warning suffices, and I don't think we should be carrying history like this.